### PR TITLE
Add .html / .css / linters / images

### PR DESCRIPTION
**Pull request by J2ZROMERO**
> During the project. 😀
- Use CSS GRID to specify columns and rows for each principal parent content (desktop and mobile).
- Responsive layout: mobile version ( < 768 px), desktop version ( > 768px).
- Apply flexbox to manage elements.
- Use Formspree in the form tag to send data to the email.
- Install linters to check the code and avoid common mistakes.
- Add class div "mobile version" (start code line: 14).
- Style mobile version (start code line: 1)
- Add folder images (github_pages\J2ZROMERO.github.io\images).
- Add enable, hover, and active events to buttons.
- Add images to the background.
- Add class div "desktop version" (start code line: 419).
- Style desktop version (start code line: 805 )
- Add folder images (github_pages\J2ZROMERO.github.io\images\desktop_version).
- Add enable, hover, and active events to buttons.
- Add images to the background.
🤩

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1 +1,61 @@
...
+- CSS
+- FLEX BOX
+- Linters
+ 
+## Getting Started
+
+**While you are navigating on my portafolio you could download,test and edit each project you choose and you like just read the instructions of each project**
+
+
+
```
Comment (HAMMAS-SALEEM) [inline README.md]: - [ ] Please add your link of deployment in the Readme.md file so it would be easily accessible to all the other people :blush:

**Review by HAMMAS-SALEEM — CHANGES_REQUESTED** [review]
# Required Changes :recycle:
almost-there

## Hey @J2ZROMERO :wave:

## Highlights :1st_place_medal:

:heavy_check_mark: Following GitHub Flow
:heavy_check_mark: Following requirements
:heavy_check_mark: Descriptive PR Summary

Good job on implementing some of the requirements, there are some issues that you still need to work on to go to the next project but you are almost there! :blush:

# Optional

_See Below Comments_

_All the changes with optional tags are not crucial enough to prevent you from getting the approval but we highly recommend to students that they should take these changes into consideration._

Cheers and Happy coding! :clap: :clap: :clap:

Feel free to leave any questions or comments in the PR thread if something is not 100% clear. Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

---------------------------------------

_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Comment by J2ZROMERO** [comment]
Added link in README.md code line #33

**Review by codepantha — APPROVED** [review]
# STATUS: APPROVED 🟢 

## Hi @J2ZROMERO 👋, 

Well done implementing the required changes 👏 
Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! 🎉 



Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**